src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -173,28 +173,29 @@ internal class CommandLineParameterParser
         private const int MaxPipePathLengthMacOS = 104;
 
         internal static string[] validParameters = {
-            "version",
-            "nologo",
-            "noexit",
 #if STAMODE
             "sta",
             "mta",
```

Comment:
Not related the PR - we'll need to review apartments after moving to .Net Core 3.0.